release-notes/6.0/known-issues.md

```diff
@@ -58,3 +58,45 @@ To fix this, edit your `PATH` environment variable to either remove the `c:\Prog
 The first time SPA apps are run, the authority for the spa proxy might be incorrectly cached which results in the JWT bearer being rejected due to Invalid issuer. The workaround is to just restart the SPA app and the issue will be resolved. If restarting doesn't resolve the problem, another workaround is to specify the authority for your app in Program.cs: `builder.Services.Configure<JwtBearerOptions>("IdentityServerJwtBearer", o => o.Authority = "https://localhost:44416");` where 44416 is the port for the spa proxy.
 
 When using localdb (default when creating projects in VS), the normal database apply migrations error page will not be displayed correctly due to the spa proxy. This will result in errors when going to the fetch data page. Apply the migrations via 'dotnet ef database update' to create the database.
+
+## Windows Desktop (Windows Forms / WPF)
+
+### Issues running applications with Windows Desktop 6.0.2
+
+Some customers are unable to run Windows Desktop (that is, Windows Forms or WPF) applications built with 6.0.200 or later .NET SDK, if the target workstation has only .NET Windows Desktop runtime 6.0.0 or 6.0.1 installed, and receieve error messages similar to the following:
```

Comment:
I'm not sure "workstation" is the right word here.

Reply:
Would this work better?
```suggestion
Some customers are unable to run Windows Desktop (that is, Windows Forms or WPF) applications built with 6.0.200 or later .NET SDK, if the target environment has only .NET Windows Desktop runtime 6.0.0 or 6.0.1 installed, and receive error messages similar to the following:
```